internal/configs/sharding/fixedschedule.go

```diff
@@ -55,43 +52,6 @@ func (s fixedSchedule) RandomnessStartingEpoch() uint64 {
 	return mainnetRandomnessStartingEpoch
 }
 
-func (s fixedSchedule) MaxTxAmountLimit() *big.Int {
-	amountBigInt := big.NewInt(mainnetMaxTxAmountLimit)
-	amountBigInt = amountBigInt.Mul(amountBigInt, big.NewInt(denominations.One))
-	return amountBigInt
-}
-
-func (s fixedSchedule) MaxNumRecentTxsPerAccountLimit() uint64 {
-	return mainnetMaxNumRecentTxsPerAccountLimit
-}
-
-func (s fixedSchedule) MaxTxPoolSizeLimit() int {
-	return mainnetMaxTxPoolSizeLimit
-}
-
-func (s fixedSchedule) MaxNumTxsPerBlockLimit() int {
-	return mainnetMaxNumTxsPerBlockLimit
```

Comment:
wouldnt we still need some throttling of this kind? This sets max tx num to 8000 per block and this logic existed before I added additional throttling

Reply:
Oh i see. Below we now hardcoded that max number of txns per block. It was 8000 before and now 1000 I guess.